fission/src/ui/panels/GraphicsSettingsPanel.tsx

```diff
@@ -32,6 +153,84 @@ const GraphicsSettings: React.FC<PanelPropsImpl> = ({ panelId, openLocation, sid
 
+    const initialPreset = detectCurrentPreset(
+        PreferencesSystem.getGraphicsPreferences().lightIntensity,
+        PreferencesSystem.getGraphicsPreferences().fancyShadows,
+        PreferencesSystem.getGraphicsPreferences().maxFar,
+        PreferencesSystem.getGraphicsPreferences().cascades,
+        PreferencesSystem.getGraphicsPreferences().shadowMapSize,
+        PreferencesSystem.getGraphicsPreferences().antiAliasing
+    )
+
+    const [selectedPreset, setSelectedPreset] = useState<string>(initialPreset)
+    const [dropdownKey, setDropdownKey] = useState<number>(0)
+
+    const updatePresetFromSettings = (
+        newLightIntensity: number,
+        newFancyShadows: boolean,
+        newMaxFar: number,
+        newCascades: number,
+        newShadowMapSize: number,
+        newAntiAliasing: boolean
+    ) => {
+        const detectedPreset = detectCurrentPreset(
+            newLightIntensity,
+            newFancyShadows,
+            newMaxFar,
+            newCascades,
+            newShadowMapSize,
+            newAntiAliasing
+        )
+        if (detectedPreset !== selectedPreset) {
+            setSelectedPreset(detectedPreset)
+            setDropdownKey(prev => prev + 1)
+        }
+    }
+
+    const applyPreset = (preset: GraphicsPreset) => {
+        const settings = GRAPHICS_PRESETS[preset]
+        const previousAntiAliasing = antiAliasing
+
+        setLightIntensity(settings.lightIntensity)
+        setFancyShadows(settings.fancyShadows)
+        setMaxFar(settings.maxFar)
+        setCascades(settings.cascades)
+        setShadowMapSize(settings.shadowMapSize)
+        setAntiAliasing(settings.antiAliasing)
+        setSelectedPreset(preset)
+        setDropdownKey(prev => prev + 1)
+
+        World.sceneRenderer.setLightIntensity(settings.lightIntensity)
+        World.sceneRenderer.changeLighting(settings.fancyShadows)
+
+        if (settings.fancyShadows) {
+            World.sceneRenderer.changeCSMSettings({
+                lightIntensity: settings.lightIntensity,
+                fancyShadows: settings.fancyShadows,
+                maxFar: settings.maxFar,
+                cascades: settings.cascades,
+                shadowMapSize: settings.shadowMapSize,
+                antiAliasing: settings.antiAliasing,
+            })
+        }
+
+        if (previousAntiAliasing !== settings.antiAliasing) {
+            setReload(true)
+            globalAddToast?.(
+                "info",
+                "Refresh Required",
+                "Anti-aliasing has been changed. Please refresh the page to see the effects."
+            )
+        }
+    }
+
+    const handleAntiAliasingChange = (checked: boolean) => {
+        setAntiAliasing(checked)
+        setReload(true)
+        updatePresetFromSettings(lightIntensity, fancyShadows, maxFar, cascades, shadowMapSize, checked)
+        globalAddToast?.("info", "Refresh Required", "Please refresh the page to see the anti-aliasing changes.")
```

Comment:
This was fixed in one of Zach's PRs right
```suggestion
        globalAddToast.("info", "Refresh Required", "Please refresh the page to see the anti-aliasing changes.")

Reply:
Assuming you mean
```suggestion
        globalAddToast("info", "Refresh Required", "Please refresh the page to see the anti-aliasing changes.")
```

Can you link the PR you are mentioning?

Reply:
#1204